packages/angular/projects/ui-angular/src/lib/components/authenticator/components/form-field/form-field.component.ts

```diff
@@ -65,6 +65,12 @@ export class FormFieldComponent implements OnInit {
     return validationError[this.name];
   }
 
+  public onBlur($event: Event) {
+    let { name } = <HTMLInputElement>$event.target;
+
+    this.authenticator.updateForm({ name: `touched_${name}`, value: true });
```

Comment:
Now we can track the touched elements in the form with the name `touched_password` or `touched_confirm_password`